hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -305,38 +305,47 @@ This option may be set to 0 to block self-update without requiring the EEPROM co
 
 Default: `1` (`0` in versions prior to 2020-09-03)  
 
-## Advanced boot modes - Network / USB mass storage boot.
-For network or USB mass storage boot we recommend updating to bootloader version 2020-09-03 and Raspberry Pi OS 2020-08-20 or newer.
-
-### Updating the bootloader
-#### Update using the Raspberry Pi Imager
-The easiest method to update the bootloader with a factory default configuration supporting USB boot is to use the Raspberry Pi Imager.
-1. Download the [Raspberry Pi Imager](https://www.raspberrypi.org/downloads/)
-1. Download the latest [EEPROM recovery image](https://github.com/raspberrypi/rpi-eeprom/blob/master/releases.md)
-1. Select `Use Custom` to reformat and flash a **blank** SD card with the EEPROM update enabling USB MSD boot support.
-
-#### Manual bootloader update
-* From a Raspberry Pi OS 2020-08-20 or newer SD card:
+## USB and network boot
+## Operating system support 
+Before using USB and network boot, it is advisable to update the operating system to the latest version. 
+
+## Selecting the boot mode
+Pi 400 and newer Raspberry Pi 4B boards support USB boot by default. In order to enable USB boot on earlier boards, or to select alternate boot modes, the bootloader configuration must be updated.
+
+### Using Raspberry Pi Imager to update the bootloader (recommended)
+Raspberry Pi Imager provides a GUI for creating SD cards image to update the bootloader
```

Comment:
"creating SD cards image to update the bootloader" -> "creating an SD card that can update the bootloader" ?